modules/oxide/src/less/skins/ui/tinymce-5/skin.less

```diff
@@ -136,6 +136,9 @@
     padding: 0;
   }
 
+  // Distraction-free mode
```

Comment:
Isn't quick toolbars available for all editor modes?

Reply:
Fair, changed it here and in the `toolbar.less`